script/Ciel/Day 11/11_01_CIEL11_1.txt

```diff
@@ -496,17 +503,18 @@ The caustic sharpness of her attitude clearly displays her unwillingness to chan
 [sha:2a1910689f710255b374c4ad079a9af24084778e]{
 -- Page 24, Offset 29067.
 -- 兄さんも今朝はそれなりですね。昨夜は夜更かしをしていたようですが」
-For once, you're quite early yourself, Nii-san. Didn't you stay up late last night?"
+You're up surprisingly early yourself.
+ I'd been under the impression you had been out late last night?"
 }
 [sha:7486a56be4d52c1ad871ba4cca9d94e48efe42ae]{
 -- Page 25, Offset 29068.
 -- ああ、それで怒っている風を装っていたのか。
-Oh, so that's why she was acting all upset.
+Ah, so that's why she's acting all upset.
 }
 [sha:5402a4760aaec2d05301e3f9557a876d4479b502]{
 -- Page 25, Offset 29069.
 -- 家の決まりを守らせる為の威圧。
-She puts on an intimidating air to guarantee that the rules of the house are followed.
+She's trying to scare me into following the rules of the house.
```

Comment:
Well, it should stand out a little. This observation is obviously a bit strange coming from Shiki, since it's clearly our favorite pal Roa making a comment here (next line too).